src/firefly/js/charts/dataTypes/FireflySpectrum.js

```diff
@@ -131,7 +132,18 @@ export function getSpectrumProps(tbl_id, spectrumDM) {
     const xErrArray = spectralAxis.statErrHigh || spectralAxis.statError;
     const xErrArrayMinus = spectralAxis.statErrLow;
     const yErrArray = fluxAxis.statErrHigh || fluxAxis.statError;
-    const yErrArrayMinus = fluxAxis.statErrLow;
+    let yErrArrayMinus = fluxAxis.statErrLow;
+    if (yErrArray && fluxAxis.statErrLow) {
+        const errCol= getColumnIdx(tbl,yErrArray);
+        const lowCol= getColumnIdx(tbl,fluxAxis.statErrLow);
+        let allEmpty=true;
+        const reverse= tbl.tableData.data.every( (r) => {
+            if (isNaN(r[errCol]) || isNaN(r[lowCol]) || r[errCol===null] || r[lowCol]===null) return true;
```

Comment:
```suggestion
            if (isNaN(r[errCol]) || isNaN(r[lowCol]) || r[errCol]===null || r[lowCol]===null) return true;
```
I think you meant this?

Reply:
yes, I will fix it.